StaticAnalyzer/views/android/find.py

```diff
@@ -15,49 +18,67 @@
 
 
 def run(request):
-    """Find in source files."""
+    """Find Filename/Content in source files."""
     try:
-        if not re.match('^[0-9a-f]{32}$', request.POST['md5']):
-            return print_n_send_error_response(request, 'Searching Failed')
+        match = re.match('^[0-9a-f]{32}$', request.POST['md5'])
+        if not match:
+            raise ValueError('Invalid MD5 hash')
         md5 = request.POST['md5']
         query = request.POST['q']
         code = request.POST['code']
-        matches = set()
```

Comment:
@matandobr This section added support for kotlin, and android zipped source code , both eclipse and android studio. This looks like removed in the PR